apps/test/util/wrappedEventListener.js

```diff
@@ -26,11 +30,15 @@ module.exports = {
    * Detach our wrapper, removing all added listeners
    */
   detach: function () {
-    window.addEventListener = original;
-    original = null;
-    listeners.forEach(function (argList) {
+    if (!window.originalAddEventListener ||
+        window.originalAddEventListener.toString().indexOf('[native code]') === -1) {
+      throw new Error('window.originalAddEventListener is not a native function');
+    }
+    window.addEventListener = window.originalAddEventListener;
+    window.originalAddEventListener = null;
```

Comment:
superminor nit: `window.originalAddEventListener = undefined;` will return us to a state slightly closer to default